diff --git a/node_modules/leaflet/dist/leaflet-src.esm.js b/node_modules/leaflet/dist/leaflet-src.esm.js
index 1bb1d6a..e8e44e4 100644
--- a/node_modules/leaflet/dist/leaflet-src.esm.js
+++ b/node_modules/leaflet/dist/leaflet-src.esm.js
@@ -59,6 +59,7 @@ var lastId = 0;
 // @function stamp(obj: Object): Number
 // Returns the unique ID of an object, assigning it one if it doesn't have it.
 function stamp(obj) {
+	if (!obj) { return undefined; }
 	if (!('_leaflet_id' in obj)) {
 		obj['_leaflet_id'] = ++lastId;
 	}
@@ -2379,6 +2380,7 @@ function create$1(tagName, className, container) {
 // @function remove(el: HTMLElement)
 // Removes `el` from its parent element
 function remove(el) {
+	if (!el) { return; }
 	var parent = el.parentNode;
 	if (parent) {
 		parent.removeChild(el);
@@ -2743,6 +2745,8 @@ var eventsKey = '_leaflet_events';
 // Removes all previously added listeners from given HTMLElement
 function off(obj, types, fn, context) {
 
+	if (!obj) { return this; }
+
 	if (arguments.length === 1) {
 		batchRemove(obj);
 		delete obj[eventsKey];
@@ -12635,6 +12639,7 @@ var Canvas = Renderer.extend({
 	},
 
 	_destroyContainer: function () {
+		if (!this._container) { return; }
 		cancelAnimFrame(this._redrawRequest);
 		delete this._ctx;
 		remove(this._container);
@@ -12642,6 +12647,7 @@ var Canvas = Renderer.extend({
 		delete this._container;
 	},
 
+
 	_updatePaths: function () {
 		if (this._postponeUpdatePaths) { return; }
 
@@ -13240,6 +13246,7 @@ var SVG = Renderer.extend({
 	},
 
 	_destroyContainer: function () {
+		if (!this._container) { return; }
 		remove(this._container);
 		off(this._container);
 		delete this._container;
@@ -13247,6 +13254,7 @@ var SVG = Renderer.extend({
 		delete this._svgSize;
 	},
 
+
 	_update: function () {
 		if (this._map._animatingZoom && this._bounds) { return; }
 
diff --git a/node_modules/leaflet/dist/leaflet-src.js b/node_modules/leaflet/dist/leaflet-src.js
index eeb30df..b958919 100644
--- a/node_modules/leaflet/dist/leaflet-src.js
+++ b/node_modules/leaflet/dist/leaflet-src.js
@@ -65,6 +65,7 @@
   // @function stamp(obj: Object): Number
   // Returns the unique ID of an object, assigning it one if it doesn't have it.
   function stamp(obj) {
+  	if (!obj) { return undefined; }
   	if (!('_leaflet_id' in obj)) {
   		obj['_leaflet_id'] = ++lastId;
   	}
@@ -2385,6 +2386,7 @@
   // @function remove(el: HTMLElement)
   // Removes `el` from its parent element
   function remove(el) {
+  	if (!el) { return; }
   	var parent = el.parentNode;
   	if (parent) {
   		parent.removeChild(el);
@@ -2749,6 +2751,8 @@
   // Removes all previously added listeners from given HTMLElement
   function off(obj, types, fn, context) {
 
+  	if (!obj) { return this; }
+
   	if (arguments.length === 1) {
   		batchRemove(obj);
   		delete obj[eventsKey];
@@ -12641,6 +12645,7 @@
   	},
 
   	_destroyContainer: function () {
+  		if (!this._container) { return; }
   		cancelAnimFrame(this._redrawRequest);
   		delete this._ctx;
   		remove(this._container);
@@ -12648,6 +12653,7 @@
   		delete this._container;
   	},
 
+
   	_updatePaths: function () {
   		if (this._postponeUpdatePaths) { return; }
 
@@ -13246,6 +13252,7 @@
   	},
 
   	_destroyContainer: function () {
+  		if (!this._container) { return; }
   		remove(this._container);
   		off(this._container);
   		delete this._container;
@@ -13253,6 +13260,7 @@
   		delete this._svgSize;
   	},
 
+
   	_update: function () {
   		if (this._map._animatingZoom && this._bounds) { return; }
 
